docs/settings/general-infra-logs-ui-settings.asciidoc

```diff
@@ -0,0 +1,17 @@
+`xpack.infra.enabled`:: Set to `false` to disabled the Logs and Infrastructure UI plugin {kib}. Defaults to `true`.
+
+`xpack.infra.sources.default.logAlias`:: Index pattern for matching indices that contain log data. Defaults to `filebeat-*`.
+
+`xpack.infra.sources.default.metricAlias`:: Index pattern for matching indices that contain Metricbeat data. Defaults to `metricbeat-*`.
+
+`xpack.infra.sources.default.fields.timestamp`:: Timestamp used to for sorting log entries. Defaults to `@timestamp`.
+
+`xpack.infra.sources.default.fields.message`:: Fields used for the message for Logs UI. Defaults to `['message', '@message']`.
+
+`xpack.infra.sources.default.fields.tiebreaker`:: Field used to break ties between two entries with the same timestamp. Defaults to `_doc`
```

Comment:
minor: lines 11, 13, 15, and 17 should have periods to make them consistent with other descriptions in this section.